### PR TITLE
Add file suggestions to testing sections in FEATURES. Fill out DINTO …

### PR DESCRIPTION
**Pull request by laurmurclar**
…Logfile Generation section in FEATURES. Fix small typos in README and BUILDING_MANUALLY

### CODE REVIEW

File: doc/FEATURES.md [hunk 1]
```diff
@@ -205,7 +207,17 @@ to production.
...
 ## DINTO Logfile Generation - Complete
 
-See [On Screen DINTO Reporting](#on-screen-dinto-reporting---complete)
+### Description
+A record of queries made to DINTO (via Chiron/Asclepius) can be found in the console logs of the Chiron service.
+
+### Testing
+This feature is tested manually.
+
+First, open up the tail of the panacea logs:
```
Comment (c-brenn) [inline doc/FEATURES.md]: do you think we should mention that the system must be running here?

**Review by c-brenn — CHANGES_REQUESTED** [review]
excellent work. some good spots 👀

File: doc/FEATURES.md (hunk 1 above)
Comment (laurmurclar) [reply doc/FEATURES.md]: Yes! Good idea. I also just noticed that i left "panacea" in from when I copy-pasted! Thanks!

File: README.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
 # Tom and the Toothfairies
 [![spacemacs badge]][spacemacs github] [![build status badge]][circle ci]
 
+*Note: While Markdown is very readable from plain text, we recommend viewing these docs on Github for convenience.*
```
Comment (c-brenn) [inline README.md]: is this supposed to be `from` => `form of`

Comment (laurmurclar) [reply README.md]: Yup!

**Review by c-brenn — APPROVED** [review]

**Comment by c-brenn** [comment]
actually, because we say `the system must be running to do this` he might assume that the logs are not persisted anywhere.

Do you think it would be worth adding something to let him know that docker-compose stores the logs on the host system too?

**Comment by laurmurclar** [comment]
@c-brenn thought about that, but the json stuff you mentioned seemed like it might confuse him a bit sinse he's so anti-docker. He seemed happy enough with just using the running service logs in the meeting we had last week, but maybe I picked up on that wrong? Up to you, I can add in the extra info no problem.

**Comment by c-brenn** [comment]
Yeah I agree, the details would be a bit much. I suppose we should wait and see - if he asks about the logs not being persisted we can then add the extra info.

**Comment by laurmurclar** [comment]
SGTM ✨

**Pull request merged**